frontend/src/container/RoutingPolicies/DeleteRoutingPolicy.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, Modal, Typography } from 'antd';
+import { Trash2, X } from 'lucide-react';
+
+import { DeleteRoutingPolicyProps } from './types';
+
+function DeleteRoutingPolicy({
+	handleClose,
+	handleDelete,
+	routingPolicy,
+	isDeletingRoutingPolicy,
+}: DeleteRoutingPolicyProps): JSX.Element {
+	return (
+		<Modal
+			className="delete-policy-modal"
+			title={<span className="title">Delete Routing Policy</span>}
+			open
+			closable={false}
+			onCancel={handleClose}
+			footer={[
+				<Button
+					key="cancel"
+					onClick={handleClose}
+					className="cancel-btn"
+					icon={<X size={16} />}
+					disabled={isDeletingRoutingPolicy}
+				>
+					Cancel
+				</Button>,
+				<Button
+					key="submit"
+					icon={<Trash2 size={16} />}
+					onClick={handleDelete}
+					className="delete-btn"
+					disabled={isDeletingRoutingPolicy}
+				>
+					Delete Routing Policy
+				</Button>,
```

Comment:
Not in this PR, but ideally, when action is in progress there should be loader in this "Delete Routing Policy" along with the disabled.

Reply:
Ok. Let me make a doc for routing policies with all such future improvements.

https://www.notion.so/signoz/Routing-Policies-Future-UX-improvements-27bfcc6bcd19807697bad12e72b204d0?source=copy_link